src/extractor.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import click
+import nltk
+
+def noun_phrases(text, *args, **kwargs):
+    sentences = (nltk.word_tokenize(sent) for sent in
+                 nltk.sent_tokenize(text))
+
+    tagged_sentences = nltk.pos_tag_sents(sentences)
+
+
+    # chunking baseado em https://www.nltk.org/book/ch07.html
+    grammar = "NP: {<DT>?<JJ.*>*<NN.*>*<JJ.*>*}"
+
+    cp = nltk.RegexpParser(grammar)
+
+    NPs = [] # Sintagmas nominais como objetos Tree
+    flat_noun_phrases = [] # Strings com os sintagmas nominais
+    for sentence_tree in cp.parse_sents(tagged_sentences):
+        for subtree in sentence_tree.subtrees():
+            if subtree.label() == "NP":
+                NPs.append(subtree)
+                np = ' '.join(w[0] for w in subtree.leaves())
+                flat_noun_phrases.append(np)
+                yield np
+
+def all_words(text, *args, **kwargs):
+    tokenizer = nltk.RegexpTokenizer(r'\w+')
```

Comment:
I think we can be more permissive here so we don't end up losing words. For example, `nexus-6` will result in 2 tokens: `nexus` and `6`, given this regex. What do you think about changing this to `[\w\d-']+`?

Reply:
I wonder if there are more special chars to be aware of besides `-` and `'`...